README.md

```diff
@@ -36,14 +37,14 @@ You can set the tasks that the crawler runs in `crawler.js`.
 
 ## Testing
 ```sh
-npm test
+gulp test
```

Comment:
Well, they still don't need to care about how the tests actually run. But as "gulp" is 1 char longer than "npm" I agree with the change :smirk: